src/Controller/ImageFormController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Image;
+use App\Form\ImageFormType;
+use App\Repository\ImageRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\File\Exception\FileException;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use App\Service\IdsListGenerator;
+
+class ImageFormController extends AbstractController
+{
+    /**
+     * @Route("/form/", name="form")
+     */
+    public function index(
+        Request $request,
+        EntityManagerInterface $entityManager,
+        ImageRepository $imageRepository,
+        IdsListGenerator $idsListGenerator
+    ): Response {
+        $image = new Image();
+        $imageForm = $this->createForm(ImageFormType::class, $image);
+
+        $imageForm->handleRequest($request);
+        if ($imageForm->isSubmitted() && $imageForm->isValid()) {
+            $imageIds = $imageForm->get('otherImages')->getData();
+            $entityManager->persist($image);
+            $entityManager->flush();
+            $imageIds = $idsListGenerator->setImageIdsList($imageIds, $image);
+            $imageIdsArr = explode(', ', $imageIds);
+            foreach ($imageIdsArr as $imageId) {
+                $images[] = $imageRepository->find($imageId);
+            }
```

Comment:
The setImageIdsList() could do this too and return the Image array directly
in a addAndGetImages() method for example

Reply:
Done